optimum/onnxruntime/training_args.py

```diff
@@ -65,6 +65,11 @@ class ORTTrainingArguments(TrainingArguments):
         metadata={"help": "The optimizer to use."},
     )
 
+    loss_in_train: Optional[bool] = field(
+        default=False,
+        metadata={"help": "Use ModuleWithLoss Wrapper to compute loss inside the training loop."},
```

Comment:
shall we make it clear when we need it? for example, when label smother is NOT none, having this will help save memory for ORTModule runs.